utils/ccloud_library.sh

```diff
@@ -766,7 +776,7 @@ function ccloud::create_ccloud_stack() {
   SCHEMA_REGISTRY_CREDS=$(ccloud::create_credentials_resource $SERVICE_ACCOUNT_ID $SCHEMA_REGISTRY)
 
   if $enable_ksql ; then
-    KSQL_NAME="demo-ksql-$SERVICE_ACCOUNT_ID"
+    KSQL_NAME=${KSQL_NAME:-"demo-ksql-$SERVICE_ACCOUNT_ID"}
     KSQL=$(ccloud::create_ksql_app $KSQL_NAME $CLUSTER)
```

Comment:
Same here, ksqlDB app names can have spaces as well.  Thank you

```suggestion
    KSQL=$(ccloud::create_ksql_app "$KSQL_NAME" $CLUSTER)
```